pkg/rdt/rdt.go

```diff
@@ -153,7 +156,6 @@ func Initialize(resctrlGroupPrefix string) error {
 
 	r := &control{Logger: log, resctrlGroupPrefix: resctrlGroupPrefix}
 
-
```

Comment:
nitpick: why the indirect assignment via the temporary `r` instead of directly to `rdt` ?

Reply:
Really no point, a refactoring leftover. Thanks for pointing this out. Fixed now